packages/ui/core-components/src/lib/unsorted/viz/funnel/_FunnelChart.svelte

```diff
@@ -169,7 +169,9 @@
 				formatter: function (params) {
 					let output;
 					if (showPercent) {
-						output = `${formatValue(params.value, valueColFormat)} (${params.percent}%)`;
+						const initialValue = data[0][valueCol];
+						const percentOfInitial = ((params.value / initialValue) * 100).toFixed(1); // one decimal place by default
```

Comment:
Let's maintain 2 decimal places in the percentage

```suggestion
						const percentOfInitial = ((params.value / initialValue) * 100).toFixed(2);
```